src/main/java/com/mide/gangsaeng/board/CacheableBoardReadStorage.java

```diff
@@ -0,0 +1,67 @@
+package com.mide.gangsaeng.board;
+
+import java.time.Duration;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Repository;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.lettuce.core.RedisException;
+import io.lettuce.core.SetArgs;
+import io.lettuce.core.api.sync.RedisCommands;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+@Repository
+public class CacheableBoardReadStorage implements BoardReadStorage {
+    private final BoardReadStorage db;
+    private final BoardReadStorage cache;
+    private final RedisCommands<String, String> redisCommands;
```

Comment:
We should not use `RedisCommands` here.
This should be used only in `BoardReadCacheStorage`.

This `CacheableBoardReadStorage` focus on **how to manipulate each cache and db storage implementatons** and provide cacheable storage layer.